src/Controller/AboutController.php

```diff
@@ -106,6 +109,12 @@ public function aimsScopeAction(Request $request) : Response
                 }
 
                 return $sections;
+            })
```

Comment:
I reckon we should get rid of this now. It was only there to prevent a completely empty page in development/testing.